catalog/spatial/wfs/spatial-wfs-mapper/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -83,6 +46,14 @@
              required="false" type="String"
              description="When Sorting By Distance, Sort By This Feature Property."/>
 
+		 <AD id="attributeMappings"
+			 name="Attribute Mappings"
+			 description='List of attribute mappings using a handlebars template. Example: {"attributeName": "ext.the-attribute", "featureName": "TheId", "template": "{{myFeature.theid}}"}'
```

Comment:
nit : we may not want to expose using extended attributes to the administrator, might be best to drop `ext.`

Reply:
removed.